recipes/ann/build.sh

```diff
@@ -0,0 +1,17 @@
+if [ "$(uname)" == "Darwin" ];
+then
+    export MACOSX_VERSION_MIN=10.9
+    export CXXFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -std=c++11"
+    export LINKFLAGS="-mmacosx-version-min=${MACOSX_VERSION_MIN}"
+    export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11 -L${LIBRARY_PATH}"
+fi
+
+
+cmake -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9 \
```

Comment:
This should use the variable instead of being hard coded:
```suggestion
cmake -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET} \
```
(Similarly above)